tests/models/nn/transformers/test_bert4rec.py

```diff
@@ -837,6 +852,7 @@ def initial_config(self) -> tp.Dict[str, tp.Any]:
             "item_net_constructor_kwargs": None,
             "pos_encoding_kwargs": None,
             "lightning_module_kwargs": None,
+            "similarity_module_kwargs": {"dist": "dot"},
```

Comment:
We should have default kwargs anywhere. But we do have default distance in DistanceSimilarityModule, so there is no need for this one here

Reply:
Fixed